<ui version="4.0" >
 <class>InterfaceDialog_q</class>
 <widget class="QWidget" name="InterfaceDialog_q" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Interface</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QFrame" name="frame3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>2</number>
        </property>
        <property name="topMargin" >
         <number>2</number>
        </property>
        <property name="rightMargin" >
         <number>2</number>
        </property>
        <property name="bottomMargin" >
         <number>2</number>
        </property>
        <property name="horizontalSpacing" >
         <number>2</number>
        </property>
        <property name="verticalSpacing" >
         <number>2</number>
        </property>
        <item row="0" column="1" >
         <widget class="QLabel" name="editorTitle" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Interface</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="pixmapLabel1" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap/>
          </property>
          <property name="scaledContents" >
           <bool>true</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame7" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>1</number>
        </property>
        <property name="topMargin" >
         <number>1</number>
        </property>
        <property name="rightMargin" >
         <number>1</number>
        </property>
        <property name="bottomMargin" >
         <number>1</number>
        </property>
        <property name="horizontalSpacing" >
         <number>2</number>
        </property>
        <property name="verticalSpacing" >
         <number>2</number>
        </property>
        <item row="0" column="3" >
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Comment:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item rowspan="3" row="1" column="3" >
         <widget class="QTextEdit" name="comment" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="tabChangesFocus" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QFrame" name="frame26" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>2</number>
           </property>
           <property name="topMargin" >
            <number>2</number>
           </property>
           <property name="rightMargin" >
            <number>2</number>
           </property>
           <property name="bottomMargin" >
            <number>2</number>
           </property>
           <property name="horizontalSpacing" >
            <number>2</number>
           </property>
           <property name="verticalSpacing" >
            <number>2</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabel1" >
             <property name="text" >
              <string>Name:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="textLabel2_2" >
             <property name="text" >
              <string>Label:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="textLabel1_2" >
             <property name="text" >
              <string>Library:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="seclevelLabel" >
             <property name="text" >
              <string>Security level:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QSpinBox" name="seclevel" >
             <property name="toolTip" >
              <string>&lt;p>Each interface of the firewall must have security level associated with it.&lt;br>Security level can be any number between 0 and 100, 0 being least secure and 100 being most secure levels. Interface with security level 0 ususally serves Internet connection.&lt;/p></string>
             </property>
             <property name="whatsThis" >
              <string>&lt;p>Each interface of the firewall must have security level associated with it.&lt;br>
Security level can be any number between 0 and 100, 0 being least secure and 100 being most secure levels. Interface with security level 0 ususally serves Internet connection.&lt;/p></string>
             </property>
             <property name="buttonSymbols" >
              <enum>QAbstractSpinBox::PlusMinus</enum>
             </property>
             <property name="maximum" >
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QLineEdit" name="label" />
           </item>
           <item row="1" column="1" >
            <widget class="QComboBox" name="libs" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="obj_name" />
           </item>
           <item row="5" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>2</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1" colspan="2" >
         <widget class="QComboBox" name="netzone" >
          <property name="toolTip" >
           <string>&lt;p>Network zone consists of hosts and networks that can be reached through this interface of the firewall. Subnet to which this interface is directly attached must be part of its network zone. Other subnets reachable by means of routing should alse be added to the network zone.
&lt;br>
If network zone for this interface consists of only one subnet, you can simply choose that network's object in the pull-down below. If your network zone should include multiple subnets, you need to create an Object Group, then put all hosts and networks which are going to be part of the network zone into that group and finally choose this group in the pull-down below.&lt;/p></string>
          </property>
          <property name="whatsThis" >
           <string>&lt;p>Network zone consists of hosts and networks that can be reached through this interface of the firewall. Subnet to which this interface is directly attached must be part of its network zone. Other subnets reachable by means of routing should alse be added to the network zone.
&lt;br>
If network zone for this interface consists of only one subnet, you can simply choose that network's object in the pull-down below. If your network zone should include multiple subnets, you need to create an Object Group, then put all hosts and networks which are going to be part of the network zone into that group and finally choose this group in the pull-down below.&lt;/p></string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="netzoneLabel" >
          <property name="text" >
           <string>Network zone:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QGroupBox" name="buttonGroup1" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string/>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <property name="horizontalSpacing" >
            <number>1</number>
           </property>
           <property name="verticalSpacing" >
            <number>1</number>
           </property>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="ext" >
             <property name="toolTip" >
              <string>&lt;p>One interface of the firewall must be marked as 'external'. This interface should be connected to the least secure network, usually the Internet.&lt;/p></string>
             </property>
             <property name="whatsThis" >
              <string>One interface of the firewall must be marked as 'external'. This interface should be connected to the least secure network, usually the Internet.</string>
             </property>
             <property name="text" >
              <string>This interface is external (insecure)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QCheckBox" name="management" >
             <property name="toolTip" >
              <string>&lt;p>Check if this interface is used for management (SNMP queries, remote policy install etc.)&lt;p></string>
             </property>
             <property name="text" >
              <string>Management interface</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" >
            <widget class="QRadioButton" name="unnumbered" >
             <property name="text" >
              <string>Unnumbered interface</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QRadioButton" name="dynamic" >
             <property name="text" >
              <string>Address is assigned dynamically</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QRadioButton" name="regular" >
             <property name="text" >
              <string>Regular interface</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QCheckBox" name="unprotected" >
             <property name="toolTip" >
              <string>Skip this interface while assigning policy rules</string>
             </property>
             <property name="text" >
              <string>Unprotected interface</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" >
            <widget class="QRadioButton" name="bridgeport" >
             <property name="text" >
              <string>Bridge port</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <tabstops>
  <tabstop>obj_name</tabstop>
  <tabstop>libs</tabstop>
  <tabstop>label</tabstop>
  <tabstop>seclevel</tabstop>
  <tabstop>management</tabstop>
  <tabstop>ext</tabstop>
  <tabstop>regular</tabstop>
  <tabstop>dynamic</tabstop>
  <tabstop>unnumbered</tabstop>
  <tabstop>bridgeport</tabstop>
  <tabstop>comment</tabstop>
  <tabstop>netzone</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bridgeport</sender>
   <signal>toggled(bool)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comment</sender>
   <signal>textChanged()</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dynamic</sender>
   <signal>toggled(bool)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ext</sender>
   <signal>clicked()</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>libs</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>management</sender>
   <signal>clicked()</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>netzone</sender>
   <signal>activated(int)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>obj_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regular</sender>
   <signal>toggled(bool)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seclevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unnumbered</sender>
   <signal>toggled(bool)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unprotected</sender>
   <signal>toggled(bool)</signal>
   <receiver>InterfaceDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
